dask_ml/decomposition/pca.py

```diff
@@ -149,21 +164,28 @@ class PCA(sklearn.decomposition.PCA):
     >>> dX = da.from_array(X, chunks=X.shape)
     >>> pca = PCA(n_components=2)
     >>> pca.fit(dX)
-    PCA(copy=True, iterated_power='auto', n_components=2, random_state=None,
-      svd_solver='auto', tol=0.0, whiten=False)
+    PCA(n_components=2)
```

Comment:
why were these changes necessary? They seem unrelated to the current pull request. Could you please revert them or explain why they're necessary?

Reply:
It really depends whether those are considered to be demonstrative examples (in doctest style) or actual doctests which are meant to pass.

If the case is the latter, then the proposed changes should probably still hold.

For example, in their original state, these doctests fail when run with:
```bash
py.test --verbose --doctest-modules dask_ml/
```

If, on the other hand, they're just meant to serve for demonstration only ([section 15](https://numpydoc.readthedocs.io/en/latest/format.html) of the `numpydoc` standard), then I'll be definitely happy to revert back these changes.

Reply:
This PR is adding one new parameter with a default value set to the current behavior. I wouldn't expect any tests to fail, and if they are I'd like to know more about what failed (a link to logs, if you have it).

Reply:
I don't think the doctests get run as part of the CI steps, and I imagine the doctests haven't been updated for quite some time (haven't checked with `git blame`) or the intention has never been that they get run as actual first-class-citizen test (but, instead, serve for demonstration).

E.g., when running against the latest `main`:
```python
$ py.test --verbose --doctest-modules dask_ml/
...

dask-ml/dask_ml/wrappers.py:92: DocTestFailure
___________________ [doctest] dask_ml.decomposition.pca.PCA ____________________
142 
143     Examples
144     --------
145     >>> import numpy as np
146     >>> import dask.array as da
147     >>> from dask_ml.decomposition import PCA
148     >>> X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
149     >>> dX = da.from_array(X, chunks=X.shape)
150     >>> pca = PCA(n_components=2)
151     >>> pca.fit(dX)
Expected:
    PCA(copy=True, iterated_power='auto', n_components=2, random_state=None,
      svd_solver='auto', tol=0.0, whiten=False)
Got:
    PCA(n_components=2)
...
```

Reply:
huh, ok I see. I'm not too familiar with that type of test, apologies. But it does seem unrelated to the change in this pull request, so I think it would be better to revert those changes here and open a separate issue / PR about the general topic of "doctest tests are failing / not run"

Reply:
I don't recall if we run doctests here or not. I wouldn't worry about that failing one. Scikit-Learn changed the repr of their objects to only show the parameters that are not the default, so the docstring changed.